app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala

```diff
@@ -95,7 +96,7 @@ object IdentificationTypeCodes {
   }
 }
 
-case class PackageInformation(typesOfPackages: Option[String], numberOfPackages: Option[Int], shippingMarks: Option[String])
+case class PackageInformation(id: UUID, typesOfPackages: Option[String], numberOfPackages: Option[Int], shippingMarks: Option[String])
```

Comment:
Would it not be simpler to defined id as a String?   Would allow us to perhaps change the id generator for new package infos on the FE to something different in future (although I do note in your tests that UUIDs can be created from any old string)

Reply:
In my opinion it is better to have it as `UUID` as it forces the client to use this type of ID.
I can see your point that having `String` is more flexible, but as you've said, you can create UUIDs from Strings with no problem.